docs/src/main/sphinx/connector/ranger.rst

```diff
@@ -0,0 +1,212 @@
+=======================
+Apache Ranger connector
+=======================
+
+.. raw:: html
+
+  <img src="../_static/img/apache_ranger.png" class="connector-logo">
```

Comment:
We should use [the latest Ranger logo](https://ranger.apache.org/ranger.jpg). We all love the elephant but anything I see with it now I associate with "outdated and slow".